storage_devices/lvm.py

```diff
@@ -186,6 +186,12 @@ def remove_filters_from_config():
         cmd = f"sed -i '/{filter_prototype_regex}/d' {lvm_config_path}"
         TestRun.executor.run(cmd)
 
+    @staticmethod
+    def set_use_devices_file(use_devices_file=False):
```

Comment:
I think the default value should be True, because False is confusing in context of setting/clearing flags